index.html

```diff
@@ -5,51 +5,224 @@
-                        </a>
-                    </div>
-                    <div class="navbar">
-                        <div></div>
-                        <div></div>
-                        <div></div>
-                    </div>
-                </nav>
-            </header>
-        </div>
-        <section>
-            <div class="body">
-                <div class="name">
-                    <h1>I’m Saikat</h1>
-                     <h2>Glad to see you!</h2>
+    <div class="container">
+        <div class="main"> 
+            <div class="header">
+                <header>
+                    <nav>
+                        <div>
+                            <a href="./index.html">
+                                <img class="logo" src="./image/logo.png" alt="">
+                            </a>
+                        </div>
+                        <div class="navbar">
+                            <div></div>
+                            <div></div>
+                            <div></div>
+                        </div>
+                    </nav>
+                </header>
+            </div>
+            <section>
+                <div class="body">
+                    <div class="name">
+                        <h2>I’m Saikat</h2>
+                        <h2>Glad to see you!</h2>
+                    </div>
+                    <section>
+                        <div class="description">
+                            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        </div>
+                    </section>
+                    <div class="contact">
+                        <h3>LET'S CONNECT</h3>
+                    </div>
+                    <div class="connect-logo">
+                        <div class="logo">
+                            <ul class="logo-div">
+                                <li><a href=""><img src="./image/twiter.svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector.png" alt=""></a></li>
+                                <li><a href=""><img src="./image/M.svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector(3).svg" alt=""></a></li>
+                                <li><a href=""><img src="./image/Vector(4).svg" alt=""></a></li>
+                            </ul>
+                        </div>
+                    </div>
                 </div>
-                <div class="description">
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </section>
+        </div>
+        <div class="work_section">
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that you change this `div` tag to a `section` tag.

## EXAMPLE
` <div class="work_section"></div>`
replaced with
 `<section class="work_section"></section>`

Reply:
Changed the same.